es-app/src/VolumeControl.cpp

```diff
@@ -32,7 +32,15 @@ VolumeControl::VolumeControl()
 	originalVolume = getVolume();
 }
 
-VolumeControl::VolumeControl(const VolumeControl & right)
+VolumeControl::VolumeControl(const VolumeControl & right):
+  originalVolume(0), internalVolume(0)
+#if defined (__APPLE__)
+    #error TODO: Not implemented for MacOS yet!!!
+#elif defined(__linux__)
+    , mixerIndex(0), mixerHandle(nullptr), mixerElem(nullptr), mixerSelemId(nullptr)
+#elif defined(WIN32) || defined(_WIN32)
+	, mixerHandle(nullptr), endpointVolume(nullptr)
+#endif
 {
```

Comment:
(The file has some other areas with bad indentation - feel free to correct the whitespace in a separate commit or I will do it after the PR).